build.gradle

```diff
@@ -24,12 +24,28 @@ ext {
 
 apply plugin: 'application'
 apply from: 'gradle/formatting.gradle'
+apply plugin: 'maven-publish'
 
 mainClassName = 'org.opensearch.sdk.ExtensionsRunner'
 
 group 'org.opensearch.sdk'
 version '1.0.0-SNAPSHOT'
```

Comment:
Not in this file, but I can't review a file you didn't change and the maven coordinates are a good point to make this comment.

Need to change the project name here:
https://github.com/opensearch-project/opensearch-sdk-java/blob/0727269bee55fe59a10df6ad624fa14cad9d4b1c/settings.gradle#L1

Reply:
Covered in this [PR](https://github.com/opensearch-project/opensearch-sdk-java/pull/96/files#diff-7f825392aa37acd1cee0c2e7b9bb7366ad6eac64f3e6cdd816e156bcb69d30deR1).